config-linux.md

```diff
@@ -685,6 +699,48 @@ The following parameters can be specified to set up seccomp:
 }
 ```
 
+### <a name="seccompstate" />The Seccomp State
+The seccomp state is a data structure passed via a UNIX socket.
+The container runtime MUST send the seccomp state over the UNIX socket as regular payload.
```

Comment:
```suggestion
The container runtime MUST send the seccomp state over the UNIX socket as regular payload serialized in JSON.
```

Reply:
Fixed, thanks!

Reply:
Hm, I've added this, but the hooks, that also receive a state in JSON, don't say it is serialized in JSON. From https://github.com/opencontainers/runtime-spec/blob/master/config.md#posix-platform-hooks:

> The state of the container MUST be passed to hooks over stdin so that they may do work appropriate to the current state of the container.

Then, [the state](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#state) says:
> When serialized in JSON, the format MUST adhere to the following pattern

So, I'm unsure if the spec says in some general way (like the way it says with unknown fields should be ignored) it should be serialized as JSON and I'm not finding it now.

Do you know?

I added it just in case, but please let me know if this is covered by some section already